### PR TITLE
chore: remove unused components

### LINKED ISSUE #784: ️🛠 - Remove old dashboard components
### Task description

Cleanup and remove unused components that were used for the old dashboard only

### Requirements

_No response_

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

Closes #784 
### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export * from './collectibles'
 export * from './developer'
-export * from './old-wallet'
+export * from './settings'
```
Comment (Tuditi) [inline packages/desktop/views/dashboard/index.ts]: This throws an error for me.

Comment (Tuditi) [reply packages/desktop/views/dashboard/index.ts]: `ERROR in ./views/dashboard/index.ts 3:0-27
Module not found: Error: Can't resolve './settings' in '/home/tuditi/bloom-labs/bloom/packages/desktop/views/dashboard'`

File: packages/desktop/views/dashboard/Dashboard.svelte [hunk 2]
```diff
@@ -22,13 +22,11 @@
     import { DashboardDrawerRouterView } from './drawers'
     import { Governance } from './governance'
     import { Wallet } from './wallet'
-    import { Settings } from '../settings'
-    import { OldWallet } from './old-wallet'
+    import { Settings } from './settings'
```
Comment (Tuditi) [inline packages/desktop/views/dashboard/Dashboard.svelte]: This throws an error.

**Review by Tuditi — CHANGES_REQUESTED** [review]
Error is thrown when opening the app due to a non-existing `settings` import.

File: packages/shared/src/lib/features/interfaces/wallet-features.interface.ts [hunk 3]
```diff
@@ -6,7 +6,6 @@ export interface IWalletFeatures extends IFeatureFlag {
         nft: IFeatureFlag
     }
     newDashboard: IFeatureFlag
```
Comment (nicole-obrien) [inline packages/shared/src/lib/features/interfaces/wallet-features.interface.ts]: should we also remove this flag?

**Review by Tuditi — APPROVED** [review]
LGTM! :fire:

**Pull request merged**